packages/store-indexer/src/postgres/createQueryAdapter.ts

```diff
@@ -13,7 +14,10 @@ import { getAddress } from "viem";
  */
 export async function createQueryAdapter(database: PgDatabase<any>): Promise<QueryAdapter> {
   const adapter: QueryAdapter = {
-    async findAll({ chainId, address, tableIds = [] }) {
+    async findAll({ chainId, address, filters = [] }) {
+      // If _any_ filter has a table ID, this will filter down all data to just those tables. Which mean we can't yet mix table filters with key-only filters.
```

Comment:
Should we add this to the changeset? Might make it easier to update in the docs later.

Reply:
I ended up making `tableId` required for now, so this comment doesn't really apply but I left it here so that we revisit this once we make `tableId` optional.